src/LayerPlan.h

```diff
@@ -607,6 +607,25 @@ class LayerPlan : public NoCopy
      */
     void addLinesByOptimizer(const Polygons& polygons, const GCodePathConfig& config, SpaceFillType space_fill_type, bool enable_travel_optimization = false, int wipe_dist = 0, float flow_ratio = 1.0, std::optional<Point> near_start_location = std::optional<Point>(), double fan_speed = GCodePathConfig::FAN_SPEED_DEFAULT);
 
+    /*!
+     * Add polygons to the g-code with monotonic order.
+     * \param polygons The lines to add.
+     * \param config The settings to print those lines with.
+     * \param space_fill_type The type of space filling used to generate the
+     * line segments (should be either Lines or PolyLines!)
```

Comment:
Thing is, this `space_fill_type` variable is not used at all at the moment. It's necessary to give it a `space_fill_type` because the `GCodePath` class stores it (and it's a constructor argument). However if you look at the usages of that `space_fill_type` member you'll find that it is only ever written by the constructor and never actually read.

This `space_fill_type` should be removed, in my opinion. It's only slowing the slicing down (albeit not very much, really). This documentation has been copied from the `addPathsByOptimizer` function. There is no real reason why it should be Lines or PolyLines since it's not actually used, and the optimizer itself supports polygons just as well.

Reply:
Ok, let's put that out of scope for the time being then. (Unless it's really easy to remove.)